docs/operation/bootstrap_backup_restore.rst

```diff
@@ -0,0 +1,49 @@
+Bootstrap node backup restore
+=============================
+This section describes a reliable way for backup and restore a **MetalK8s**
+bootstrap node.
+
+Backup
+******
+
+A backup file is generated at the end of the bootstrap.
+
+To create a new backup file you just have to run the backup script
+
+.. code::
+
+    /srv/scality/metalk8s-X.X.X/backup.sh
+
+Backup archive are stored in `/var/lib/metalk8s/` directory
+
+Restore
+*******
+
+.. warning::
+
+    You can't use the restore script if you not have HA apiserver as SSH
+    informations to reconfigure all nodes are stored into apiserver.
+
+.. warning::
+
+    In case of 3 members etcd cluster (2 nodes + unreachable old bootstrap node)
+    you need to remove the old bootstrap node from the etcd cluster before
+    running the restore script.
+
+To restore a bootstrap node you need a backup file and **MetalK8s** ISOs.
+
+The ISOs need to be at the same place as the old bootstrap node, according to
```

Comment:
Not sure this paragraph is very clear.

Do you mean something like:

> All the ISOs referenced in the bootstrap configuration file (located at `/etc/metalk8s/bootstrap.yaml`) must be present

?

Reply:
Yes I will update